exe/Rad.hs

```diff
@@ -0,0 +1,64 @@
+{-# LANGUAGE TypeApplications #-}
+-- | Top-level radicle CLI. Calls sub-commands.
+module Rad
+    ( main
+    ) where
+
+import           Protolude
+
+import           Data.List (isPrefixOf)
+import           Data.String (String)
+import qualified Data.Text as T
+import           System.Directory
+import qualified System.Posix.Process as Posix
+
+import           Paths_radicle (getBinDir)
+
+-- | All command filenames are prefixed with this.
+radPrefix :: String
+radPrefix = "rad-"
+
+runCommand :: [String] -> IO ()
+runCommand [] = do
+    bin <- getBinDir
+    cmds <- availableCommands [bin]
+    putStr @Text "usage: rad <command> [<args>]\n\n"
+    if null cmds then do
+        putStrLn @Text "No available rad commands."
```

Comment:
Maybe it would make sense to list the `version` and `help` commands here?

Reply:
Yeah, for some reason `git help -a` doesn't list `version` but it does list `help`.

Reply:
85fdcc3